tests/functional_tests/0003_create_test.py

```diff
@@ -21,30 +21,235 @@
 # STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
 # IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
-import re
+import os
+import json
+import subprocess
+import tempfile
+import uuid
 
 import pytest
-from click.testing import CliRunner
 
-import iocage_cli as ioc
 
 require_root = pytest.mark.require_root
 require_zpool = pytest.mark.require_zpool
+require_dhcp = pytest.mark.require_dhcp
+require_networking = pytest.mark.require_networking
+
+# LIST OF FLAGS in create TO TEST
+# [c, C , r, t, p, n, u, b, T, e, s, f]
+
+# TODO: Test different jail props as well
+
+
+@require_root
+@require_zpool
+def test_01_create_jail_with_uuid(release, jails, invoke_cli):
+    u_flag = str(uuid.uuid4())
```

Comment:
We should catch this and return a descriptive exception

Reply:
I am sorry, I am not sure what we should catch here 😅 ? Could you please elaborate ?